live-examples/js-examples/date/date-sethours.js

```diff
@@ -3,7 +3,7 @@ event.setHours(20);
 
 console.log(event);
 // expected output: "Tue Aug 19 1975 20:15:30 GMT+0200 (CEST)"
-// (note: your timezone may vary)
+// (Note: your timezone may vary)
```

Comment:
Do we need the brackets here? and all the similar instances of this "Note" in other files